src/web/app/components/logs-table/logs-table.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { LogsTableRowModel } from './logs-table-model';
+
+/**
+ * A table to display logs.
+ */
+@Component({
+  selector: 'tm-logs-table',
+  templateUrl: './logs-table.component.html',
+  styleUrls: ['./logs-table.component.scss'],
+})
+export class LogsTableComponent implements OnInit {
+
+  @Input()
+  logs: LogsTableRowModel[] = [];
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  expandDetails(logsTableRowModel: LogsTableRowModel): void {
+    logsTableRowModel.isDetailsExpanded = !logsTableRowModel.isDetailsExpanded;
+  }
+
+  getClassForStatus(httpStatus: number): string {
+    const num: number = Math.floor(httpStatus / 100);
+    if (num === 2) {
```

Comment:
Suggest to use switch case instead. We never know if we're going to need to display 1XX or 3XX kind of logs in the future.

Reply:
Alright! Will make the necessary changes